maven-spy/pom.xml

```diff
@@ -67,8 +67,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <release>8</release>
```

Comment:
last and final point, I read all your comments, I got your point of view, but please switch this to `<release>6</release>`  and the PR title to "restore Java < 11 compatibility"

I just do not want to hide a compatibility drop between a regression fix, even if no care about this drop because it is already dead and frozen
I do not know when the other PR about Java 6 drop will be merged (because there are more important before), and with a second PR we will have a second entry in the changelog, which will be clearer from my point of view

Reply:
well if you think it's important to have in the **SAME** release notes two lines saying something in a contradiction such:
- restore java 6 compatibility
- drop java 6/7 support

I hope users will understand...
Anyway as long as PRs get merged and released....... 
I'm out of arguments at this stage and don't want to try to find new one....

Reply:
that will not be in the same release !
This PR is tagged "bugfix", it will build a release just with its changes

Reply:
(To change the subject to something more pleasing… 😅) When is said release scheduled? Is the CI going to build it on its own in the coming hours?

Reply:
Yes, it is all automatic, the CI will do the job on its own soon